packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -672,9 +695,18 @@ export function createProjectionNode<I>({
          * the next step.
          */
         updateProjection = () => {
+            // Reset projection frame data
+            projectionFrameData.totalNodes =
+                projectionFrameData.resolvedTargetDeltas =
+                projectionFrameData.recalculatedProjection =
+                    0
```

Comment:
Is this significantly faster? Because it reads weird.

Reply:
Not faster but no object creation per frame to reduce GC